src/legacy/core_plugins/kibana/common/tutorials/tutorial_schema.js

```diff
@@ -70,6 +70,7 @@ const instructionVariantSchema = Joi.object({
 
 const instructionSetSchema = Joi.object({
   title: Joi.string(),
+  callOut: Joi.object(),
```

Comment:
@nreese Is this okay with you?

Reply:
Adding callOut to the schema makes since to provide messages for cases like this. I think the implementation needs a little tweaking. The schema definition should not be so tightly couple with the UI. 

So instead of defining the contents like 

```
callOut: {
    title: 'Important: Updating to 7.0 or higher',
    children: 'Please make sure your APM Server is updated to 7.0 or ....',
}
```

how about 

```
callOut: {
    title: 'Important: Updating to 7.0 or higher',
    messsages: [
        'Please make sure your APM Server is updated to 7.0 or ....',
    ]
}
```

Reply:
Works for me 👍